### PR TITLE
Try to fix long runs by not upwinding

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR:
 - Tries to fix the long runs (#409) by not upwinding
 - Removes the low frequency output, since the output is visually difficult to follow
 - Adds `fps` CL arg and sets `fps = 30` for long run animations because they're slow and take time to review

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -39,25 +39,13 @@ steps:
...
-
-      - label: ":computer: held suarez (ρe_int)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoeint --dt_save_to_sol 8640000" # 1200 days
-        artifact_paths: "longrun_hs_rhoeint/*"
-        agents:
-          slurm_cpus_per_task: 8
-
-      - label: ":computer: held suarez (ρe)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoe_largerdt --dt_save_to_sol 21600" # 1200 days
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoe_largerdt --dt_save_to_sol 21600" # 1200 days
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:42]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoe --dt_save_to_sol 21600" # 1200 days
```
I think it's just standard dt (400s)?

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -39,25 +39,13 @@ steps:
...
         artifact_paths: "longrun_hs_rhoe_largerdt/*"
         agents:
           slurm_cpus_per_task: 8
 
       - label: ":computer: held suarez (ρe_int)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoeint_largerdt --dt_save_to_sol 21600" # 1200 days
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoeint_largerdt --dt_save_to_sol 21600" # 1200 days
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:48]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoeint --dt_save_to_sol 21600" # 1200 days
```

**Review by szy21 — APPROVED** [review]

File: .buildkite/longruns/pipeline.yml (hunk 1 above)
Comment (charleskawczynski) [reply .buildkite/longruns/pipeline.yml:42]: Huh, yeah, I think that was when I was testing the larger dt, I forgot to switch it back, thanks!

**Pull request merged**